Source/main.py

```diff
@@ -54,7 +54,7 @@ def main():
             filename = getROM(romdir)
 
         # Start PyBoy and run loop
-        pyboy = PyBoy(sys.argv[1] if len(sys.argv) > 1 else None, scale, filename, bootrom)
+        pyboy = PyBoy(sys.argv[1] if len(sys.argv) > 1 else None, filename, scale, bootrom)
```

Comment:
We cannot change the interface without warning, as everyone else using PyBoy will then have to change their code.